pkg/api/datacenter_cache.go

```diff
@@ -0,0 +1,113 @@
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"sync"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type DatacenterCache struct {
+	client HTTPClient
+	token  string
+
+	mtx sync.Mutex
+	dcs []datacenter
+}
+
+type datacenter struct {
```

Comment:
latitude and longitude would be nice for doing map based dashboards, ala https://grafana.com/grafana/plugins/grafana-worldmap-panel/ or the like

Reply:
hmm, highlighting didn't work how I had hoped, meant to highlight the full datacenter struct 3 lines down, but hopefully still makes sense ;)

Reply:
I forget how that plugin works. Does it need separate lat/on metrics as values? Or can it use label strings as the values?

Reply:
I'm guessing values, probably directly as latitude & longitude

Reply:
If it's values, that means adding two new static timeseries per datacenter. That doesn't seem like a cost worth the benefit. It's gotta be possible with labels, right?

Reply:
Gah!  I believe you're right, and I suspect I might have transposed that in my head.

Reply:
Cool, this basically works out-of-the-box with the geomap panel, just set format to table.

![Screen Shot 2021-11-18 at 4 19 00 PM](https://user-images.githubusercontent.com/114509/142443374-ea6a7906-e6e0-4a2f-a43a-e2b5cd8b2a05.png)

Reply:
Awesome!